assemblies/resources/bin/setenv

```diff
@@ -35,4 +35,9 @@
 # export KARAF_REDIRECT    # Enable/set the std/err redirection when using bin/start
 
 export EXTRA_JAVA_OPTS="-Dorg.eclipse.jetty.server.Request.maxFormContentSize=1500000 -Dfile.encoding=UTF-8"
+export JAVA_MAX_MEM=1G
+export JAVA_PERM_MEM=256M
```

Comment:
Are `JAVA_PERM_MEM`, `JAVA_MAX_PERM_MEM`, and `KARAF_REDIRECT` still used? I can't find any references in any of the `bin` files.

Reply:
`JAVA_MAX_MEM` is used in `inc` and defaults to `512M`, so setting it to `1G` is probably better.